Test Results - Average Loss: 0.5151, Average Dice Score: 0.7595100%||[Finetuning] Epoch: 1 Avg Loss: 0.3132[Validation] Avg Loss: 0.3313, Avg Dice Score: 0.8904100%[Finetuning] Epoch: 2 Avg Loss: 0.3045[Validation] Avg Loss: 0.3294, Avg Dice Score: 0.8908100%[Finetuning] Epoch: 3 Avg Loss: 0.2984[Validation] Avg Loss: 0.3296, Avg Dice Score: 0.8910100%[Finetuning] Epoch: 4 Avg Loss: 0.2958[Validation] Avg Loss: 0.3301, Avg Dice Score: 0.8894100%[Finetuning] Epoch: 5 Avg Loss: 0.2926[Validation] Avg Loss: 0.3256, Avg Dice Score: 0.8968100%[Finetuning] Epoch: 6 Avg Loss: 0.2905[Validation] Avg Loss: 0.3287, Avg Dice Score: 0.8901100%[Finetuning] Epoch: 7 Avg Loss: 0.2904[Validation] Avg Loss: 0.3254, Avg Dice Score: 0.8954100%[Finetuning] Epoch: 8 Avg Loss: 0.2874[Validation] Avg Loss: 0.3294, Avg Dice Score: 0.8890100%[Finetuning] Epoch: 9 Avg Loss: 0.2879[Validation] Avg Loss: 0.3277, Avg Dice Score: 0.8891100%[Finetuning] Epoch: 10 Avg Loss: 0.2862[Validation] Avg Loss: 0.3290, Avg Dice Score: 0.8921 /home/mikhelson/Med143/143 [00:14<00:00, 9.97it/s]143/143 [00:14<00:00, 10.00it/s]143/143 [00:14<00:00, 10.08it/s]143/143 [00:14<00:00, 9.86it/s]143/143 [00:14<00:00, 10.13it/s]143/143 [00:14<00:00, 10.11it/s]143/143 [00:14<00:00, 10.19it/s]143/143 [00:14<00:00, 10.03it/s]143/143 [00:14<00:00, 10.03it/s]143/143 [00:14<00:00, 10.09it/s]ImSeg-Lab24/CLUE/lib/python3.8/site-packages/lightning/pytorch/core/module.py:447: You are trying to `self.log()` but the 'self.trainer reference is not registered on the model yet. This is most likely because the model hasn't been passed to the 'Train Test Results - Average Loss: 0.6463, Average Dice Score: 0.7208er'